application/actions/watches_test.go

```diff
@@ -0,0 +1,95 @@
+package actions
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/gobuffalo/nulls"
+
+	"github.com/silinternational/wecarry-api/domain"
+	"github.com/silinternational/wecarry-api/internal/test"
+	"github.com/silinternational/wecarry-api/models"
+)
+
+type watchQueryFixtures struct {
```

Comment:
```suggestion
type watchFixtures struct {
```
and keep the original `watchQueryFixtures` untouched. "Query" isn't a word I would normally associate with a REST API.

Reply:
:+1: